bootstrap4/templates/bootstrap4/index.js

```diff
@@ -5,6 +5,7 @@ import './components/afObjectField/afObjectField.js'
 import './components/quickForm/quickForm.js'
 
 // Input types
+import './inputTypes/autocomplete/autocomplete.html'
```

Comment:
Ok, got it. I was following the pattern from other .js to html piggybacks. Thanks!